src/cdn_definitions/schema.yaml

```diff
@@ -11,7 +11,211 @@ $id: http://release-engineering.github.io/cdn-definitions/schema.json
 
 
 definitions:
+  arch:
+    type: string
+    pattern: "^(x86|x86\\_64|ia64|i386|ppc|ppc64|ppc64le|s390|s390x|aarch64)$"
+
+  arch_list:
+    type: array
+    items:
+      $ref: "#/definitions/arch"
+
+  release_version:
+    type: string
+    pattern: "^[0-9]+\\.[0-9]+$"
+
+  major_version:
+    type: string
+    pattern: "^[0-9]+$"
```

Comment:
Should this be declared as an integer?

Reply:
The jsonschema documentation states that object keys must be a string [1], and I created this definition to be used as an object key.  I will instead create a major_version_string definition (of type string) to be used solely as an object key/property name, and convert the major_version to type integer for all other uses (outside of a property name).